server/sonar-server/src/main/java/org/sonar/server/permission/PermissionService.java

```diff
@@ -76,6 +77,11 @@ public PermissionService(DbClient dbClient, PermissionRepository permissionRepos
     return GlobalPermissions.ALL;
   }
 
+  public UserWithPermissionQueryResult findUsersWithPermission(PermissionQuery query) {
```

Comment:
The same way, it may be interesting to have a PermissionDaoFacade which is used as the entry point to several dao concerning permissions. I tried a massive refactoring and it was a pain.